mw/auth/options.go

```diff
@@ -55,17 +55,21 @@ func WithCallback(attr attributer) option {
 // to Themis in the authorization request. Specifically, this includes the gRPC
 // service name (e.g. AddressBook) and the corresponding function that is
 // called by the client (e.g. ListPersons)
-func WithRequest() option {
+func WithRequest(appID string) option {
+	// assume PARGs are in default namespace if no appID is provided
```

Comment:
@markinos - are we still equating namespace with application? I don't think we are. Which means that:

1) these comments are wrong
2) pargs need to include the application as a criterion

I think I need to think this through thoroughly and come up with a better appID story.

Reply:
> I think I need to think this through thoroughly and come up with a better appID story.

Does that mean we should we hold off on this pull request altogether?

Reply:
ok. that works for now. I will write something up on Monday, I have a few thoughts around how we can make this all work better.